membership-attribute-service/app/models/ApiErrors.scala

```diff
@@ -34,4 +34,10 @@ object ApiErrors {
     details = "Failed to authenticate",
     statusCode = 401
   )
+
+  val forbidden = ApiError(
```

Comment:
should be `val invalidApiKey` as forbidden is just the code used not the meaning